### DISCUSSION TITLE
KEDA Chart 2.6.2

### DISCUSSION THREAD

**Discussion by zroubalik** [kedacore/charts#248] [Announcements]
*Chart: 2.6.2 | App: 2.6.1*

## TL;DR

```shell
helm repo add keda https://kedacore.github.io/charts
helm install keda/keda
```

### Features
- Support for KEDA v2.6.1 ([release notes](https://github.com/kedacore/keda/releases/tag/v2.6.1))
- Add extra container args config for both KEDA Operator and Metric Server([#245](https://github.com/kedacore/charts/pull/245) -  Contributed by @marco2704 🎉 )

## New Contributors
* @marco2704 made their first contribution in https://github.com/kedacore/charts/pull/245

**Full Changelog**: https://github.com/kedacore/charts/compare/v2.6.1...v2.6.2

<hr /><em>This discussion was created from the release <a href='https://github.com/kedacore/charts/releases/tag/v2.6.2'>KEDA Chart 2.6.2</a>.</em>
